### ISSUE TITLE
🌙 Dark mode

### ISSUE THREAD

**Issue by jackyzha0** [jackyzha0/ctrl-v#63]
'nuff said

AC:
- [ ] Add toggle to switch themes
- [ ] Persist option to local storage

**Comment by Siddhant-K-code** [comment]
I can work on this!

**Comment by Siddhant-K-code** [comment]
I saw there is a lot of usage of colors on diff pages, we can add it all under once in `app.js` ??

[This](https://css-tricks.com/a-dark-mode-toggle-with-react-and-themeprovider/#:~:text=import%20React%20from%20%27react%27%3B,h1%3EIt%27s%20a%20light%20theme!) May Help

**Comment by jackyzha0** [comment]
ctrl-v actually uses styled-components under the hood and we use a Theme Provider! Code here: https://github.com/jackyzha0/ctrl-v/tree/master/frontend/src/theme

Probably just a toggle to switch the themes in here. Will add a colour palette soon

**Comment by jackyzha0** [comment]
> ctrl-v actually uses styled-components under the hood and we use a Theme Provider! Code here: [https://github.com/jackyzha0/ctrl-v/tree/master/frontend/src/theme](https://github.com/jackyzha0/ctrl-v/tree/master/frontend/src/theme?rgh-link-date=2021-04-12T16%3A37%3A34Z)
> 
> Probably just a toggle to switch the themes in here. Will add a colour palette soon

updated

**Comment by Siddhant-K-code** [comment]
It is looking this with new color pallete: 
![image](https://user-images.githubusercontent.com/55068936/114508851-763ea380-9c52-11eb-9248-1197ceae16a4.png)

## 

![image](https://user-images.githubusercontent.com/55068936/114508979-9ec69d80-9c52-11eb-8b4b-437aa72862f0.png)



will try to add a toggle, localStorage, and will try to send PR asap

**Comment by jackyzha0** [comment]
> It is looking this with new color pallete:
> ![image](https://user-images.githubusercontent.com/55068936/114508851-763ea380-9c52-11eb-9248-1197ceae16a4.png)
> 
> ![image](https://user-images.githubusercontent.com/55068936/114508979-9ec69d80-9c52-11eb-8b4b-437aa72862f0.png)
> 
> will try to add a toggle, localStorage, and will try to send PR asap

any possibility we can change the default theme of the editor to be atom dark when dark mode toggle is turned on too?

**Comment by Siddhant-K-code** [comment]
Yes, and I was also thinking of providing options at the top right of the editor for various themes :)
